### DISCUSSION TITLE
URL을 쓰는 이유는 무엇일까요?

### DISCUSSION THREAD

**Discussion by sa02045** [prgrms-web-devcourse/We-are-the-MASTERS-OF-HTTP#21] [Interview Question]
URL을 쓰는 이유는 무엇일까요?

**Comment by Hyevvy** [comment]
URL(Uniform Resource Locator)이란 인터넷의 리소스를 가리키는 표준이름이다. URL을 쓰는 이유는 전자정보 일부를 가리키고 그것이 어디에 있고 어떻게 접근할 수 있는 지 알려주기 위함이다.

**Comment by punchdrunkard** [comment]
URL은 "리소스에 접근하는데 필요한 정보"를 "하나의 인터페이스로 나타내는 방법"이라고 할 수 있습니다.  
이렇게 리소스를 일관된 방식으로 칭함으로써, 다양한 프로토콜에 대하여 리소스에 접근할 수 있습니다.

**Comment by yunjjeongjo** [comment]
URL은 인터넷에서 정보의 위치를 알려주는 표준 주소체계입니다.
웹페이지의 정보 구조가 제대로 반영된 URL은 사용자가 원하는 정보의 위치와 종류를 파악할 수 있도록 도와줍니다.

**Comment by punchdrunkard** [comment]
- (책 30p) 프로토콜과 리소스의 위치를 한번에 나타내기 위함.
